src/menu-button/_MenuButton.jsx

```diff
@@ -54,6 +56,7 @@ const MenuButton = ({
         >
           {items.map(item => (
             <Menu.Item
+              className={cx(theme.className)}
```

Comment:
put theme.className at the top of the component

Reply:
I tried putting theme.className in the React.Fragment tag, the Menu tag, and the Menu.ItemGroup tag. Anytime it's removed from the Menu.Item tags the text for the menu items fails to theme. Is it possible that I'm wiring things up wrong?

Reply:
If you wrap it with a `div` instead of `React.Fragment`, it should work. Also theme context changes for menu-item got merged yesterday, which might have impacted themes not getting applied properly.

Reply:
Good call, the update to menu-item means that menu-button is now theming without any intervention necessary in terra-dev-site.